tinygrad/tensor.py

```diff
@@ -216,6 +216,19 @@ def kaiming_normal(*shape, a:float = 0.01, **kwargs) -> Tensor:
     std = math.sqrt(2.0 / (1 + a ** 2)) / math.sqrt(prod(shape[1:]))
     return Tensor.normal(*shape, mean=0.0, std=std, **kwargs)
 
+  @staticmethod
+  def multinomial(p: Tensor, num_samples: int, replacement: bool = False) -> Tensor:
+    assert p.ndim <= 2, "p must be 1 or 2 dim"
+    assert replacement, "supported only with replacement"
```

Comment:
If num_samples is 1, is replacement=false okay?

Reply:
yep, I'll allow it